code/game/turfs/open.dm

```diff
@@ -355,3 +355,9 @@
 	if(wet_overlay)
 		cut_overlay(wet_overlay)
 
+
+/turf/open/rad_act(pulse_strength)
+	if (air.gases[/datum/gas/carbon_dioxide])
+		air.gases[/datum/gas/carbon_dioxide][MOLES]-=(pulse_strength/100)
```

Comment:
could possibly result in negatives.

Reply:
This makes co2 engine setups impossible as it'll convert all the co2 quite quickly